tests/containers/docker.pm

```diff
@@ -50,6 +51,7 @@ sub run {
     test_seccomp();
     allow_selected_insecure_registries(runtime => 'docker');
 
+    check_docker_firewall() if (check_runtime_version(get_docker_version(), ">=20.10") && $self->firewall() eq 'firewalld');
```

Comment:
I think it would be nice to add some explanation why you want to run firewall check in versions greater than `20.10`